wandb/wandb_torch.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, history):
         self._num_bins = 64
         self._is_cuda_histc_supported = None
         self._jupyter_run = None
+        self.hook_torch = TorchGraph.hook_torch
```

Comment:
What's this for?

Reply:
This is so that we can access it through `wandb.run.history.torch.hook_torch` instead of `wandb.wandb_torch.TorchGraph.hook_torch`.
This let the operation be a `noop` in disabled mode, which was not the case when going through `wandb.wandb_torch`.